ci/pipeline.yml

```diff
@@ -130,7 +130,7 @@ jobs:
       IAM_KEYS_HOST: ((aws-iam-keys-host))
       IAM_KEYS_PORT: ((aws-iam-keys-port))
       IAM_CREATE_TABLES: ((aws-iam-create-tables-bool))
-      GATEWAY_HOST: prometheus-production.service.cf.internal
+      GATEWAY_HOST: prometheus-staging.service.cf.internal
```

Comment:
I assume this is not intentional and we want to send the updated metrics to production?

Reply:
Yep, that is correct. I'm doing more work on this and will probably remove this PR, but I will double check that I updated the GATEWAY on the next PR. Thanks!